test/cmd/policy.sh

```diff
@@ -98,6 +98,34 @@ os::cmd::expect_success_and_not_text 'oc policy can-i --list --user harold --gro
 os::cmd::expect_success_and_text 'oc policy can-i --list --user harold --groups system:authenticated' 'create get.*buildconfigs/webhooks'
 
 
+os::cmd::expect_failure 'oc policy scc-subject-review'
+os::cmd::expect_failure 'oc policy scc-review'
+os::cmd::expect_failure_and_text 'oc policy scc-subject-review -f ${OS_ROOT}/test/testdata/pspreview_unsupported_statefulset.yaml' 'error: StatefulSet "rd" with spec.volumeClaimTemplates currently not supported.'
```

Comment:
missing tests for mutually exclusive option combinations.  Maybe that should go in a unit test, I know we don't have a ton of them for cli commands but we should at least make sure our validations are holding up.